### ISSUE TITLE
Still support Racket 6.9, for awhile?

### ISSUE THREAD

**Issue by greghendershott** [Bogdanp/setup-racket#23]
Thanks again for doing this!! As I know from maintaining `travis-racket` for so many years, no good deed goes unpunished. :smile: So...

I'm building/testing Racket Mode for its minimum supported version of Racket -- 6.9 -- as well as the latest release and the snapshot.

I've been using your v0.8 for awhile now, and updating the latest version of Racket, manually.

I recently updated it to 8.0. Then today I thought, heck, I should use your new `stable` label.

So I updated all the way to your v1.0. But then [I got a 404 error for Racket 6.9](https://github.com/greghendershott/racket-mode/actions/runs/568938659).

So I rolled back to your v0.12, the version just after you added `stable`. Success! Both `6.9` and `stable` worked.

I'm not sure where between v0.12 and v1.0 the support for Racket 6.9 stopped working? Also not sure if that happened intentionally or accidentally?

---

TL;DR: I actually have no practical problem, right now. Someday I might, if I needed to update to your v1.0 (or newer) for other reasons. So this issue is just a heads-up, to point out the dropping of Racket 6.9 and see what you think. Casual and not urgent.

**Comment by Bogdanp** [comment]
Thanks for the report! 

I don't think the code that deals with URLs changed in between those versions so the likely problem is that the default `variant` for 1.0 changed form BC to CS. Since there's no CS 6.9, that would explain the 404 (I need to remember to change it to print the URLs it tries to download). Can you try setting `variant: BC` for 6.9 to see if that solves the problem under 1.0?

**Comment by greghendershott** [comment]
1. Would it be possible for `variant` to default to `BC` < 8.0 and `CS` >= 8.0?  (Matching how Racket handles the default download for end users?)

2. I ask because I'm not sure how to keep this matrix while having `variant` end up `BC` for `6.9` but `CS` for `stable`:

```yaml
      matrix:
        emacs_version:
          - '25.1'              # our minimum supported version
          - '26.3'
          - '27.1'              # most recent release
        racket_version:
          - '6.9'               # our minimum supported version
          - 'stable'            # most recent release
        include:
          - emacs_version: 'snapshot'
            racket_version: 'current'
```

(I've never found documentation that has helped me really grok how matrixes work beyond the simplest cases.)

3. As a final fallback: Since there aren't that many permutations, I suppose I could just list each manually using `include:`. That wouldn't be horrible.

**Comment by Bogdanp** [comment]
> Would it be possible for variant to default to BC < 8.0 and CS >= 8.0? (Matching how Racket handles the default download for end users?)

This seems like the best approach so I just changed it to work this way. I released the change as v1.1 so you should now be able to upgrade. Let me know if you run into any other issues!

**Comment by greghendershott** [comment]
This wonderful -- thank you!
